pkg/controller/cdapmaster/cdapmaster_controller.go

```diff
@@ -275,7 +278,7 @@ func getResourceName(r *alpha1.CDAPMaster, resource string) string {
 func getPreUpgradeResources(s *upgradeJobSpec, rsrclabels map[string]string) ([]reconciler.Object, error) {
 	ngdata := &upgradeValue{Job: s}
 	ngdata.Labels = rsrclabels
-	item, err := k8s.ObjectFromFile(templateDir + upgradeJobTemplate, ngdata, &batchv1.JobList{})
+	item, err := k8s.ObjectFromFile(templateDir+upgradeJobTemplate, ngdata, &batchv1.JobList{})
```

Comment:
leave the spaces around "+"? https://github.com/cdapio/cdap-operator/pull/9#discussion_r306081445

Reply:
I need to figure out how to make the IDE auto formatter works correctly. It supposed to use gofmt.

Reply:
I'll just let gofmt do the job.